### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by gamesminer**
task-2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        <li class="food__item"><img src="img/food/mexican.jpg" alt="mexican"
+            class="food__img img"></li>
+        <li class="food__item"><img src="img/food/chinese.jpg" alt="chinese"
+            class="food__img img"></li>
+        <li class="food__item"><img src="img/food/pizza.jpg" alt="pizza" class="food__img img"></li>
+        <li class="food__item"><img src="img/food/american.jpg" alt="american"
+            class="food__img img"></li>
+      </ul>
+      <ul class="food__statistics menu">
+        <li class="food__statistics_item">
```
Comment (ihor4568) [inline index.html]: modifier can't be used itself - it should be applied to either block or element.

File: scss/components/_features.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.features {
+  background-color: #fff;
+
+  &__heading {
+    margin: 2.5rem auto 2rem;
+    width: 15rem;
+
+    &_burger::before {
```
Comment (ihor4568) [inline scss/components/_features.scss]: burger, money and chef -- these should better to be elements.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+<body>
+
+  <header class="header">
+    <div class="header__container container">
+      <div class="header__subsection">
+        <h1><a href="#" class="header__logo"><img src="img/header/logo.png" alt="logo"
+              class="header__img"></a></h1>
+        <div class="header__facilities">
+          <a href="#" class="link header__subsection_item header__log_in">log
```
Comment (ihor4568) [inline index.html]: where is the element for the `header__log_in` modifier? Maybe you should just use smth like element `header__log-in ` without modifier

File: index.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+            in</a>
+          <a href="#" class="link header__subsection_item header__log_up">sign
```
Comment (ihor4568) [inline index.html]: same as ^^

**Comment by ihor4568** [comment]
You use modifiers too much - in most of the cases you used them - you could use just one more element. And sometimes you forget to put an element or block for modifier. Remember that modifier can only modify smth - so it can not be put just on tag itself

**Comment by gamesminer** [comment]
> You use modifiers too much - in most of the cases you used them - you could use just one more element. And sometimes you forget to put an element or block for modifier. Remember that modifier can only modify smth - so it can not be put just on tag itself

Thank you. Fixed)

**Pull request merged**